Added simple graphql app

Arfey opened the pull request:
Sorry, for my estimate 😞

Now, i added structure for graphql api. It's chat because i want to show all possible of `GraphQl` (query, mutation, subscribes).

`contrib.py` - it's file that solve problem in few libs. I going to make pull request in these libs for remove that file in future.

todo:
 -- add correct ui (now ui has problems)
 -- add test
 -- add authorization

exapmle:
<img width="1482" alt="screen shot 2018-05-28 at 22 29 01" src="https://user-images.githubusercontent.com/10367100/40627639-c61db88a-62c8-11e8-88ca-07e71d4aad70.png">

jettify commented:
I suggest not to add authorization in this PR, it is already huge and requires some time to read.

jettify commented on demos/graphql/README.rst:
```diff
@@ -0,0 +1,5 @@
+pip install -r requirements-dev.txt
```
Probably good idea to update README early with information how to start app, as well as purpose of the app.

Arfey commented:
Okay. I will add full `ui` and `readme`. I will wait for comments 😊

jettify commented:
Lots of code, not sure I can review properly. But I do not mind merging if we have simple tests for most of handlers.

webknjaz commented:
Also needs rebasing.

webknjaz commented:
You probably didn't package it correctly and it didn't make it into dist.

webknjaz commented:
Packaging it into an installable distribution requires knowledge of `setuptools` specifics (like manifest) and actual *need* of having it. (Yes, I'm still against putting `setup.py` every here and there).

Arfey commented:
So, we have green build. ✅
I moved config files to package, also removed `pytest-asyncio` from my tests, because it's has conflict with tests in other projects. It's so bad. In future we should isolate requirements and run tests for different projects. 🤔

jettify commented:
There are minor things here and there, but those if required can be addressed in separate PRs. As for packaging, I do not think that is big problem. We can address that later too.
